R/gs_power_wlr.R

```diff
@@ -208,12 +209,21 @@ gs_power_wlr <- function(enroll_rate = define_enroll_rate(duration = c(2, 2, 10)
     interval = interval
   )
 
+  # set up H1 spending
+  if (h1_spending) {
+    theta1 <- x$theta
+    info1 <- x$info
+  } else {
+    theta1 <- 0
+    info1 <- x$info0
+  }
+
   # Given the above statistical information calculate the power ----
   y_h1 <- gs_power_npe(
-    theta = x$theta,
+    theta = x$theta, theta0 = 0, theta1 = theta1,
```

Comment:
Since all the other arguments are on their own line, I recommend keeping this consistent with the surrounding code and moving `theta0` and `theta1` to new lines